### ISSUE TITLE
with_maximized(true) produces non-maximized window

### ISSUE THREAD

**Issue by eterevsky** [tomaka/winit#471]
OS: Windows 10 Home, Version 1708, build 16299.371
winit version: 0.11.3
GPU: Nvidia GTX 1080, driver 385.54
Scaling factor: 150%

```rust
 let surface = winit::WindowBuilder::new()
        .with_maximized(true)
        .build_vk_surface(&events_loop, instance.clone())
        .unwrap();
```

opens a non-maximized window with dimensions 1024x768 instead of a maximized window.

**Comment by francesca64** [comment]
Do you have this problem running off of master? This was implemented recently: #457

**Comment by eterevsky** [comment]
Nope, I'm picking up the latest version from Cargo, 0.11.3. Should it be included in that release? `set_fullscreen` and `set_decorations` work for me, though the former sets wrong resolution (#469).

Let me try to link the version from master.

**Comment by eterevsky** [comment]
Ah, yes. I'm playing with Vulkano and I can't try the latest version because the latest vulkano-win doesn't work with winit 0.12 or later.

**Comment by francesca64** [comment]
Thanks. We should have a new release out pretty soon.
